openfecwebapp/templates/partials/candidates-filter.html

```diff
@@ -19,7 +19,7 @@
       <ul>
         <li>
           <input id="cycle-checkbox-five-thousand" name="five_thousand_flag" type="checkbox" value="true">
-          <label for="cycle-checkbox-five-thousand">Has raised more than $5,000</label>
+          <label for="cycle-checkbox-five-thousand">Has raised more than $0</label>
```

Comment:
In addition to the above change, the `for` attribute should change as well (`cycle-checkbox-has-raised-funds`).